app/controllers/ems_infra_controller.rb

```diff
@@ -68,6 +68,10 @@ def scaling
       return_message = _("Scaling")
       @count_parameters.each do |p|
         if !scale_parameters[p.name].nil? && scale_parameters[p.name] != p.value
+          if scale_parameters[p.name].to_s < p.value.to_s
+            log_and_flash_message(_("Scaling down is not supported. New value for %{name} %{new_value} is lower than current value %{current_value}.") % {:name => p.name, :new_value => scale_parameters[p.name], :current_value => p.value})
```

Comment:
`log_and_flash_message()` was removed recently from our code. Now, you should just use `add_flash(...)`

Reply:
@mzazrivec OK I updated this to reflect the change.